Makefile

```diff
@@ -142,11 +178,11 @@ unit-xml: clean
 	ECK_TEST_LOG_LEVEL=$(LOG_VERBOSITY) gotestsum --junitfile unit-tests.xml -- -cover ./pkg/... ./cmd/... $(TEST_OPTS)
 
 integration: GO_TAGS += integration
-integration: clean generate-crds
+integration: clean generate-crds-v1beta1
```

Comment:
Afaik I know we cannot until we upgrade kubebuilder. Because we are relying on the kubebuilder integration test support and kubebuilder 2.3.1 comes with a 1.15 control plane. I can add another comment to make this clear

Reply:
TIL, and my bad: you already mentioned it [here](https://github.com/elastic/cloud-on-k8s/pull/4489/files#diff-e5ec31770015c79fe55d4b0deded05848e0d54c6f1ddc74cfc02e9c32854e735R31).